views/js/qtiCreator/editor/gridEditor/droppable.js

```diff
@@ -28,18 +28,28 @@ define([
     "use strict";
     var droppableGridEditor = {};
 
-    droppableGridEditor.createDroppableBlocks = function createDroppableBlocks(qtiClass, $el, options){
+    var _pulseTimer;
+    var _pulse;
 
-        options = options || {};
+    droppableGridEditor.createDroppableBlocks = function createDroppableBlocks(qtiClass, $el, optionsOriginal){
+        var options = optionsOriginal || {};
 
-        var minUnits = (typeof options.min === 'string') ? options.min : 0,
-            $colInitial = (options.initialPosition instanceof $) ? options.initialPosition : null,
-            ns = options.namespace ? '.' + options.namespace : '',
-            data = options.data || {};
+        var minUnits = (typeof options.min === 'string') ? options.min : 0;
+        var $colInitial = (options.initialPosition instanceof $) ? options.initialPosition : null;
+        var ns = options.namespace ? '.' + options.namespace : '';
+        var data = options.data || {};
 
-        var $placeholder = $('<div>', {'id' : 'qti-block-element-placeholder', 'class' : 'qti-droppable-block-hover'}),
-            marginWidth = parseFloat($el.find('[class^="col-"]:last, [class*=" col-"]:last').css('margin-left')),
-            isEmpty = ($el.children('.grid-row').length === 0);
+        var $placeholder = $('<div>', {'id' : 'qti-block-element-placeholder', 'class' : 'qti-droppable-block-hover'});
+        var marginWidth = parseFloat($el.find('[class^="col-"]:last, [class*=" col-"]:last').css('margin-left'));
+        var isEmpty = ($el.children('.grid-row').length === 0);
+
+        var _appendPlaceholder;
```

Comment:
I don't think using this type of var is the TAO way. `_appendPlaceholder`.
There are a lot in this file.
 Can you change it please?

Reply:
Developers are still using the dash to defined that variable is used privately.